Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -60,6 +60,7 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
         // static constexpr amrex::Real Ms = 1e4;
         // Real constexpr cons1 = -mag_gamma_interp; // should be mu0*gamma, mu0 is absorbed by B used in this case
         // Real constexpr cons2 = -cons1*alpha/Ms; // factor of the second term in scalar LLG
+        Real M_normalized_error = 0.1; // normalization error of M field for checking
```

Comment:
in a future PR we should make M_normalized_error a parameter we can set via the inputs file (e.g., "mag_M_normalized_error") in the MacroscopicProperties class.